guides/common/modules/proc_configuring-project-with-keycloak-for-totp-authentication.adoc

```diff
@@ -0,0 +1,32 @@
+[id="configuring-project-with-keycloak-for-totp-authentication_{context}"]
+= Configuring {Project} with {Keycloak} for TOTP Authentication
+
+Use this procedure to configure {Project} to use {Keycloak} as an OpenID provider for external authentication with Time-based One-time Password (TOTP).
+
+.Prerequisites
+
+* {Project} configured with {Keycloak} external authentication.
+
+.Procedure
+
+. In the {Keycloak} web UI, navigate to the {Project} realm.
+. Click *User Federation* and click the user federation you created for {Project}.
+. Click the *Mappers* tab.
+. Click the *Create* button to add a group mapper.
+. In the *Name* field, enter the name for the {Project} group mapper.
+. From the *Mapper Type* list, select *group-ldap-mapper*.
+. In the *LDAP Groups DN* field, enter the Group DN from your LDAP server.
+. From the *User Groups Retrieve Strategy* list, select *LOAD_GROUP_BY_MEMBER_ATTRIBUTE*.
+. Click *Save*.
+. Click each mapper and set the *Always Read Value From LDAP* setting to *ON*.
+. Navigate to *Authentication*, and click the *OTP Policy* tab.
+. Ensure that the *Supported Applications* field includes FreeOTP or Google Authenticator.
+. Configure the OTP settings to suit your requirements.
+. Optional: If you want to use TOTP authentication as a default authentication method for all users, click the *Flows* tab, and to the right of the *OTP Form* setting, select *REQUIRED*.
+. Click the *Required Actions* tab.
+. To the right of the *Configure OTP* row, select the *Default Action* check box
+. To verify that OTP authentication works, log in to {Project}, {Project} redirects you to SSO login screen.
+. Enter your username and password, and click *Log In*.
+. Because this is the first attempt to log in, {Keycloak} requests you to configure your client by scanning the barcode and entering the pin displayed.
+. After configuring your client and entering a valid PIN, you are redirected to {Project} and
+logged in.
```

Comment:
This must be a part of the title: `Configuring User Federation for Satellite in Red Hat Single Sign-On`

Reply:
Rahul's comment is about moving steps 2-9 to the User Federation proc